RPGManager_development/src/rpg_database/character_sheet/Background.java

```diff
@@ -69,6 +88,7 @@ public void setSelfInSheet(CharacterSheet characterSheet) {
 					Fields.BASECLASS).toString()));
 		}
 		characterSheet.characterData.put(Fields.BACKGROUND, this);
+		characterSheet.setData(Fields.LANGUAGES, this.language);

```

Comment:
"hasonló elvű program" - mondanál egy példát?
"ezt szoktam meg, hogy ha base classot, backgroundot állítasz akkor resetel minden eddig beállított dolgot" - ha átállítasz egy base class-t vagy egy backgroundot, akkor megnézi hogy az általad indítványozott új base class/background kompatibilis -e a többi beállítással és ha nem az akkor hibát dob. Egyéb esetben pedig csak az módosul. Tehát nem értem hogy mire gondolsz pontosan, légyszi fejtsd ki.

És ok hogy a defaultokat át lehet állítani, csak idegesítő. Ha a kiajánlott default language felülcsapja a user által addig beállítottakat az a usernek idegesítő lesz, azt nem akarom. Akkor inkább oldjuk majd meg hogy frontend logikával feldob egy message boxot hogy "Do you want to add this language <backgroundDefaultLanguage>?" és akkor most ne is foglalkozz ezzel az extra viselkedéssel.

Vagy csak simán adja hozzá background váltásnál az adott új background default nyelvét ahhoz a sethez ami tartalmazza a már eddig hozzáadott nyelveket, hiszen ebben állapodtunk meg amikor ez a téma anno szóba került.